README.md

```diff
@@ -1,3 +1,6 @@
+> [!CAUTION]
+> **pii_safe_schema** has been deprecated. We use Satori to perform data masking for our database schemas.
```

Comment:
Could say "is being deprecated" and then we can update it once we archive the repo, but I'm good either way :D

Reply:
Fair enough!